community/examples/hpc-slurm-ubuntu2004.yaml

```diff
@@ -24,11 +24,10 @@ vars:
   instance_image:
```

Comment:
Please don't declare `instance_image` as a deployment variable.

Reply:
Why? This seems like a good way to make sure the same image is applied to all modules (compute, controller, login).

Reply:
Declaring `instance_image` as a deployment variable causes errors when it propagates to resources it isn't intended for (such as DDN). The current work-around I discussed with Ivan is renaming the deployment variable `instance_image` to some thing else and then calling it explicitly in the modules.